cc/scripts/scripts.js

```diff
@@ -38,6 +39,16 @@ const CONFIG = {
   lcpImg?.removeAttribute('loading');
 }());
 
+async function loadArticlePromotions() {
+  if (!(document.querySelector('main .promotion') instanceof HTMLElement)) {
+    const promotionElement = document.querySelector('head meta[name="promotion"]');
+    if (promotionElement) {
+      const { default: decoratePromotion } = await import('../features/short-article-promotion.js');
+      decoratePromotion(promotionElement);
+    }
+  }
+}
```

Comment:
```suggestion
async function loadArticlePromotions(getMetadata) {
  const promoEl = document.querySelector('main .promotion');
  const promoMeta = getMetadata('promotion');
  if (!(promoEl && promoMeta)) return;
  const { default: decoratePromo } = await import('../features/short-article-promotion.js');
  decoratePromo(promoEl);
}
```

We like early returns. I'm not advocating you **_must_** do the above, but it's another way you can write this that is a bit cleaner. I could argue with myself:

1. Is passing in getMetadata really that much better or needed?
1. Should we really check for promoMeta before we know if a promo is on the page?

Reply:
I'd probably also remove the word "short" from "short-article-promotion" as what does "short" actually mean? Pretty much nothing in this context... we're not checking for any sort of lengths. I think this is bad hold over from an early Franklin project.

Reply:
@auniverseaway I have integrated the mentioned changes from review comment.